internal/datacoord/services.go

```diff
@@ -115,12 +115,7 @@ func (s *Server) AssignSegmentID(ctx context.Context, req *datapb.AssignSegmentI
 			zap.Uint32("count", r.GetCount()))
 
 		if coll := s.meta.GetCollection(r.CollectionID); coll == nil {
```

Comment:
Shall we give up query root when collection is not found here?

Reply:
> Shall we give up query root when collection is not found here?

oh I should call s.GetCollection()